tools/platform/BUILD

```diff
@@ -7,15 +7,7 @@ config_setting(
 )
 
 config_setting(
-    name = "ubuntu",
-    values = {
-        "cpu": "k8",
-    },
-    visibility = ["//visibility:public"],
-)
-
-config_setting(
-    name = "centos",
+    name = "k8",
```

Comment:
I am wondering if the change in this file is expected/needed in this PR. it seems unrelated and looks a little strange.

Reply:
good catch it is not needed.

Reply:
@nwangtw - i need to take my comment back. this is needed to ensure all linux variants are grouped under one target which is k8 - otherwise, it is ubuntu, centos and more variations.

Reply:
I see. So now k8 is considered as a platform? A little unclear but not a blocker to me.